azure-proto-compute/VirtualMachineContainer.cs

```diff
@@ -101,21 +103,21 @@ public async override Task<ArmOperation<VirtualMachine>> StartCreateAsync(string
         /// <summary>
         /// Construct an object used to create a VirtualMachine.
         /// </summary>
-        /// <param name="vmName"> The name of the Virtual Machine. </param>
+        /// <param name="hostName"> The hostname for the virtual machine. </param>
         /// <param name="adminUser"> The admin username to use. </param>
         /// <param name="adminPw"> The admin password to use. </param>
         /// <param name="nicId"> The network interface id to use. </param>
         /// <param name="aset"> The availability set to use. </param>
         /// <param name="location"> The location to create the Virtual Machine. </param>
```

Comment:
When location = null. We should clearly indicate where the default value will be coming from.

Reply:
this has an upcoming ado item to track https://dev.azure.com/azure-mgmt-ex/DotNET%20Management%20SDK/_workitems/edit/5333